### ISSUE TITLE
Add distinct x and y projections (or a log-log projection)

### ISSUE THREAD

**Issue by adigitoleo** [GenericMappingTools/GMT.jl#664]
When trying to copy the second GMT tutorial into GMT.jl I have encountered a missing symbol (I think). The GMT command I want to reproduce is:

```
gmt begin GMT_tut_2
	gmt basemap -R1/10000/1e20/1e25 -JX9il/6il -Bxa2+l"Wavelength (m)" -Bya1pf3+l"Power (W)" -BWS
gmt end show
```

Both the x and y axis are in a log scale. At the moment, this can be achieved by specifying the `l` flag to the `figsize` strings:

```julia
function tutorial2()
    basemap(
        region = (1, 10000, 1e20, 1e25),
        figsize = ("9il", "6il"),
        frame = (
            axes = (:left_full, :bottom_full),
            xlabel = "Wavelength (m)",
            ylabel = "Power (W)",
        ),
        xaxis = (annot = 2, ticks = :auto),
        yaxis = (annot = "1p", ticks = 3),
        show = true,
        Vd = 1,
    )
end
```

However, it would be nice to separate projection and size. Maybe there could be distinct `xproj` and `yproj` arguments (larger change, would require rework of existing projection bindings) or just a `:logxy` projection. Let me know which one of these you prefer, and I will look into an implementation (unless you already have something planned).

**Comment by joa-quim** [comment]
Hmm, ``proj`` can take the values ``logx, logy, loglog, powx, powy``

E.g. from the tests file
```
basemap(region=(1,1000,0,1), proj=:logx, figsize=(8,0.7), frame=(annot=1, ticks=2, grid=3, scale=:pow), show=1)
```

**Comment by adigitoleo** [comment]
Aha! yes, the `:loglog` is what I needed. Thanks. I can't see it in the documentation for `proj`, although the linked GMT docs eventually revealed the `l` option to `-J`. Maybe the linear aliases that you listed can be added to the table in [the proj docs](https://www.generic-mapping-tools.org/GMT.jl/dev/common_opts/#proj)?

**Comment by joa-quim** [comment]
Yes, I fished them in the code 🙁. Made a small commit that added them to ``proj`` docs but I fear it's not enough. GMT splits the _log_ and _power_ plots controls between ``proj`` (**-J**) and ``axes`` (**-B**) which is not obvious to condense in a nice simple syntax. This is an area where we clearly need testing and examples ... feel free to.

**Comment by adigitoleo** [comment]
I see, I'll have to table this until I get more time to explore these options, but I'll keep this open for discussion.
